data/simpy.gram

```diff
@@ -45,10 +45,17 @@ continue_stmt: a='continue' { _Py_Continue(EXTRA(a, a)) }
 import_stmt: import_name | import_from
 import_name[stmt_ty]: a='import' b=dotted_as_names { _Py_Import(b, EXTRA(a, b)) }
 # note below: the ('.' | '...') is necessary because '...' is tokenized as ELLIPSIS
-import_from: ('from' (('.' | '...')* !'import' dotted_name | ('.' | '...')+)
-              'import' ('*' | '(' import_from_as_names ')' | import_from_as_names))
-import_from_as_names: import_from_as_name (',' import_from_as_name)* [',']
-import_from_as_name: NAME ['as' NAME]
+import_from[stmt_ty]:
+    | a='from' b=('.' | '...')* !'import' c=dotted_name 'import' d=import_from_targets { _Py_ImportFrom(((expr_ty) c)->v.Name.id, d, seq_count_dots(b), EXTRA(a, d)) }
```

Comment:
There's a hack when the action gets uncomfortably long: put the `{` at the end of one line, and put the rest of the action on the next line, with some indentation. I think it would serve us well here.